components/ui/form/radio-area/RadioAreaGroup.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { PropsWithChildren, useState } from "react";
+import React, { ReactNode, useState } from "react";
 
 import classNames from "@lib/classNames";
 
-type RadioAreaProps = React.InputHTMLAttributes<HTMLInputElement> & {
-  onChange: (value: string) => void;
-  defaultChecked: boolean;
+type RadioAreaProps = Omit<React.InputHTMLAttributes<HTMLInputElement>, "onChange"> & {
+  onChange?: (value: string) => void;
+  defaultChecked?: boolean;
```

Comment:
This whole file was a tricky one to fix types

Reply:
Yeah, I think it needs 

- deprecating
- refactoring a copy to be more compliant to normal `<input`-fields
- moved into where the `<Input/>`

 (not as part of this PR)